src/greeting.js

```diff
@@ -0,0 +1,16 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { fetchGreeting } from './features/greeting/greetingSlice';
+
+function Greeting() {
+  const greeting = useSelector((state) => state.greeting);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(fetchGreeting());
+  }, [dispatch]);
+
+  return <h1>{greeting}</h1>;
+}
+
+export default Greeting;
```

Comment:
- [ ] [OPTIONAL] It is a good practice to always have the components in a component directory. This will help you keep an organized root directory.